library/std/src/sys/pal/unix/thread_parking.rs

```diff
@@ -1,3 +1,7 @@
+// Only used on NetBSD. If other platforms start using id-based parking, use
+// separate modules for each platform.
+#![cfg(target_os = "netbsd")]
```

Comment:
Hmm... I'm not sure I love this being `unix/thread_parking.rs` with nothing in the path to indicate it's highly platform specific. If more unix targets do use id-based parking, what should they call their modules? How would that be organised?

Reply:
Then I'd split them `unix/thread_parking/netbsd.rs`/`unix/thread_parking/*.rs` and expose the relevant inferface in `unix/thread_parking/mod.rs`.